integration-tests/testing-framework/src/test/java/io/quarkiverse/githubapp/it/testingframework/TestingFrameworkTest.java

```diff
@@ -68,6 +67,35 @@ void ghObjectVerify() {
                         "GHIssue#750705278.addLabels(\"otherValue\");");
     }
 
+    @Test
+    void ghObjectVerifyNoMoreInteractions() {
+        ThrowingCallable assertion = () -> when().payloadFromClasspath("/issue-opened.json")
+                .event(GHEvent.ISSUES)
+                .then().github(mocks -> {
+                    verify(mocks.issue(750705278))
+                            .addLabels("someValue");
+                    verifyNoMoreInteractions(mocks.ghObjects());
+                });
+
+        // Success
+        IssueEventListener.behavior = (payload, configFile) -> {
+            payload.getIssue().addLabels("someValue");
+        };
+        assertThatCode(assertion).doesNotThrowAnyException();
+
+        // Failure
+        IssueEventListener.behavior = (payload, configFile) -> {
+            payload.getIssue().addLabels("someValue");
+            payload.getIssue().addLabels("otherValue");
+        };
+        assertThatThrownBy(assertion)
+                .isInstanceOf(AssertionError.class)
+                .hasMessageContainingAll("No interactions wanted here:",
+                        "-> at io.quarkiverse.githubapp.it.testingframework.TestingFrameworkTest.lambda$ghObjectVerifyNoMoreInteractions$",
+                        "But found this interaction on mock 'GHIssue#750705278':",
+                        "-> at io.quarkiverse.githubapp.it.testingframework.TestingFrameworkTest.lambda$ghObjectVerifyNoMoreInteractions$");
```

Comment:
Do we have something about using `addLabels` after this line? Because it's going to be hard to figure out what's going on.

Reply:
We don't, and any complains should go to Mockito. That's not my doing.

Reply:
FWIW:

* the error includes the line number.
* the errors are formatted in such a way that an IDE will turn the line reference into a hyperlink, allowing you to jump to the call site and see which method was called.